src/main/java/com/company/TAM2/Main.java

```diff
@@ -0,0 +1,47 @@
+package com.company.TAM2;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+
+    public static final int COLUMNS_COUNT = 2;
+
+    public static void main(String[] args) {
+        String fromFileName =  "orange.csv";
```

Comment:
please also add the "orange.csv" file to the PR, put it into `src/main/resources` folder

Reply:
fixed